Fedora/input/checks/chronyd_specify_remote_server.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
+  <definition class="compliance" id="chronyd_specify_remote_server" version="1">
+    <metadata>
+      <title>Specify a Remote NTP Server for Time Data</title>
+      <affected family="unix">
+        <platform>multi_platform_rhel</platform>
```

Comment:
This `<platform>` check should not exist unless this check is for both RHEL 6&7 and Fedora and should be in `shared/oval` if this check is RHEL is supported.